src/Service/Users.php

```diff
@@ -119,7 +119,7 @@ public function getUserData($username, $authToken, array $options = []) {
         ],
       ];
 
-      $apiOptions = $this->config->get('attributes');
+      $apiOptions = $this->config['attributes'];
```

Comment:
Please can you rename `$this->config` to something a little more descriptive? Its not the entire "config", its only operations, therefore a more logical and descriptive name would be:
`$this->openamApiOperations`

Reply:
Updated.